gallery/src/styles/NxBtn/NxBtnPage.tsx

```diff
@@ -145,6 +153,13 @@ const NxBtnPage = () =>
       error buttons. Note that the standard disabled styles override the error styles.
     </GalleryExampleTile>
 
+    <GalleryExampleTile title="Small"
+                        htmlExample={nxBtnSmallCode}
+                        codeExamples={nxBtnSmallCode}>
+      A demonstration of an <NxCode>nx-btn</NxCode> using "small" styles, along with a disabled
+      error buttons. Note that the standard disabled styles override the small styles.
```

Comment:
What does it mean for the standard disabled styles to override the small styles? (I know this is similar wording to the above, but if the small style is overridden that reads like it'll become normal sized :sweat_smile: )

Reply:
Well it looks fine on the page, just the wording is a bit odd
![image](https://github.com/user-attachments/assets/de5c80c7-275c-4fd1-a4bb-3d5442a7aa1b)